bash_functions.sh

```diff
@@ -22,14 +22,20 @@ prepare_configs() {
     # Re-apply perms from basic-install over any volume mounts that may be present (or not)
     # Also  similar to preflights for FTL https://github.com/pi-hole/pi-hole/blob/master/advanced/Templates/pihole-FTL.service
     chown pihole:root /etc/lighttpd
-    chown pihole:pihole "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" "/var/log/pihole" "${regexFile}"
+    chown pihole:pihole "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" "/var/log/pihole" 
     chmod 644 "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf"
     # not sure why pihole:pihole user/group write perms are not enough for web to write...dirty fix:
-    chmod 777 "${regexFile}"
+    if [[ -n ${regexFile} && -e ${regexFile} ]]; then 
```

Comment:
Don't think we need both checks. `-e` should be enough, if the VAR is empty then the file won't exist.